### PR TITLE
TACOSTRAT-453 Basic CQL Measure Execution Endpoint

### PR DESCRIPTION
**Pull request by okeefm**
This PR is for the "raw results" calculation work done by myself, Tom, Matt, and Chris. 

NOTE: To make this work locally, switch the `cql-exec-fhir` and `cql-execution` dependencies to local deps (and download the branches indicated by package.json locally). The dependencies should look like below:

```
"cql-exec-fhir": "file:../cql-exec-fhir",
"cql-execution": "file:../cql-execution"
```

TODO: add tests

### CODE REVIEW

**Comment by mgramigna** [comment]
DoD says we should also add an endpoint in https://github.com/projecttacoma/fqm-execution-service for the calculateRaw functionality, so I think a companion PR there is warranted. It should be pretty simple, as the app is all set up. Just need to pass measure and patient into the calculate call [here](https://github.com/projecttacoma/fqm-execution-service/blob/master/src/app.ts). For now perhaps we should also rename this endpoint to calculateRaw

File: script.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { R4 } from '@ahryman40k/ts-fhir-types';
```
Comment (mgramigna) [inline script.ts:1]: do we still need this now that we have the cli?

File: src/Execution.ts [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  measureBundle: R4.IBundle,
+  patientBundles: R4.IBundle[],
+  options: CalculationOptions
+): {
+  rawResults?: cql.Results;
+  errorMessage?: string;
+  elmLibraries?: ELM[];
+  mainLibraryName?: string;
+} {
+  // TODO: return type^
```
Comment (mgramigna) [inline src/Execution.ts:21]: Obsolete TODO?

File: src/Execution.ts [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      libraries.push(library);
+      const elmsEncoded = library.content?.filter(a => a.contentType === 'application/elm+json');
+      elmsEncoded?.forEach(elmEncoded => {
+        if (elmEncoded.data) {
+          const decoded = Buffer.from(elmEncoded.data, 'base64').toString('binary');
+          const elm = JSON.parse(decoded) as ELM;
+          if (library.id === rootLibId) {
+            rootLibIdentifer = elm.library.identifier;
+          }
+          // This line is a hack to
```
Comment (mgramigna) [inline src/Execution.ts:51]: ...

File: src/Execution.ts (hunk 2 above)
Comment (okeefm) [reply src/Execution.ts:21]: Yeah, looks that way

File: src/Execution.ts (hunk 3 above)
Comment (okeefm) [reply src/Execution.ts:51]: 😕

File: script.ts (hunk 1 above)
Comment (okeefm) [reply script.ts:1]: Probably not

File: src/Execution.ts (hunk 2 above)
Comment (hossenlopp) [reply src/Execution.ts]: I think I meant it as a question along the lines as "should we externalize the definition to an `interface`".

Comment (okeefm) [reply src/Execution.ts]: Ah. I thought it was a comment we'd made when we were returning an `any` here before

Comment (mgramigna) [reply src/Execution.ts]: Ah, that makes sense. i think we can move this return type to one of our type declaration files, yeah

Comment (okeefm) [reply src/Execution.ts]: Yeah, I'd be fine with that change too

File: src/DebugHelper.ts [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import { ELM } from './types/ELMTypes';
+import { ValueSetMap } from 'cql-execution';
+import fs = require('fs');
```
Comment (mgramigna) [inline src/DebugHelper.ts:3]:
```suggestion
import fs from 'fs';
```

Comment (okeefm) [reply src/DebugHelper.ts]: Updated. You may have to clean away your branch and re-pull, since I had to amend the commit before this one.

**Review by mgramigna — DISMISSED** [review]
ez

**Review by mgramigna — APPROVED** [review]

**Pull request merged**